src/main/java/it/gov/pagopa/payhub/activities/service/ingestionflow/debtpositiontype/DebtPositionTypeProcessingService.java

```diff
@@ -0,0 +1,130 @@
+
+import com.opencsv.exceptions.CsvException;
+import it.gov.pagopa.payhub.activities.connector.debtposition.DebtPositionTypeService;
+import it.gov.pagopa.payhub.activities.connector.organization.OrganizationService;
+import it.gov.pagopa.payhub.activities.dto.ingestion.debtpositiontype.DebtPositionTypeErrorDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.debtpositiontype.DebtPositionTypeIngestionFlowFileDTO;
+import it.gov.pagopa.payhub.activities.dto.ingestion.debtpositiontype.DebtPositionTypeIngestionFlowFileResult;
+import it.gov.pagopa.payhub.activities.mapper.ingestionflow.debtpositiontype.DebtPositionTypeMapper;
+import it.gov.pagopa.payhub.activities.service.ingestionflow.IngestionFlowProcessingService;
+import it.gov.pagopa.pu.debtposition.dto.generated.CollectionModelDebtPositionType;
+import it.gov.pagopa.pu.debtposition.dto.generated.DebtPositionType;
+import it.gov.pagopa.pu.organization.dto.generated.Organization;
+import it.gov.pagopa.pu.processexecutions.dto.generated.IngestionFlowFile;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Service
+@Lazy
+@Slf4j
+public class DebtPositionTypeProcessingService extends
+    IngestionFlowProcessingService<DebtPositionTypeIngestionFlowFileDTO, DebtPositionTypeIngestionFlowFileResult, DebtPositionTypeErrorDTO> {
+
+  private final DebtPositionTypeMapper debtPositionTypeMapper;
+  private final DebtPositionTypeService debtPositionTypeService;
+  private final OrganizationService organizationService;
+
+
+  public DebtPositionTypeProcessingService(
+      DebtPositionTypeMapper debtPositionTypeMapper,
+      DebtPositionTypeErrorsArchiverService debtPositionTypeErrorsArchiverService,
+      DebtPositionTypeService debtPositionTypeService, OrganizationService organizationService) {
+    super(debtPositionTypeErrorsArchiverService);
+    this.debtPositionTypeMapper = debtPositionTypeMapper;
+    this.debtPositionTypeService = debtPositionTypeService;
+    this.organizationService = organizationService;
+  }
+
+
+  public DebtPositionTypeIngestionFlowFileResult processDebtPositionType(
+      Iterator<DebtPositionTypeIngestionFlowFileDTO> iterator,
+      List<CsvException> readerException,
+      IngestionFlowFile ingestionFlowFile, Path workingDirectory) {
+
+    List<DebtPositionTypeErrorDTO> errorList = new ArrayList<>();
+    DebtPositionTypeIngestionFlowFileResult ingestionFlowFileResult = new DebtPositionTypeIngestionFlowFileResult();
+    ingestionFlowFileResult.setDebtPositionTypeCodeList(new ArrayList<>());
+
+    Optional<Organization> organizationBroker = organizationService.getOrganizationById(
+        ingestionFlowFile.getOrganizationId());
+    Organization organization = organizationBroker.orElse(null);
```

Comment:
why perform this on 2 instructions?
```suggestion
    Organization organization = organizationService.getOrganizationById(
        ingestionFlowFile.getOrganizationId()).orElse(null);
```